$OpenBSD: patch-src_standardfeedsview_cpp,v 1.1 2014/02/07 16:07:12 dcoppa Exp $

Remove obsolete categories in the Browse tab
(upstream git commit b298cf2309ea8b3b94fe58b720bf1bff68dd8c98)

--- src/standardfeedsview.cpp.orig	Thu Dec 19 00:37:27 2013
+++ src/standardfeedsview.cpp	Fri Feb  7 16:46:53 2014
@@ -95,19 +95,14 @@ void StandardFeedsView::addVideoSourceWidget(VideoSour
     connect(w, SIGNAL(activated(VideoSource*)),
             SIGNAL(activated(VideoSource*)));
     int i = layout->count();
-    static const int cols = 5;
+    static const int cols = 4;
     layout->addWidget(w, i / cols, i % cols);
 }
 
 QList<YTStandardFeed*> StandardFeedsView::getMainFeeds() {
     QList<YTStandardFeed*> feeds;
 
-    feeds << buildStardardFeed("most_popular", tr("Most Popular"))
-          // << buildStardardFeed("recently_featured", tr("Featured"))
-          << buildStardardFeed("most_shared", tr("Most Shared"))
-          << buildStardardFeed("most_discussed", tr("Most Discussed"))
-          << buildStardardFeed("top_rated", tr("Top Rated"))
-          << buildStardardFeed("most_popular", tr("All Time Popular"), "all_time");
+    feeds << buildStardardFeed("most_popular", tr("Most Popular"));
 
     return feeds;
 }
